contents/blog/analytics-pricing.md

```diff
@@ -0,0 +1,157 @@
+---
+date: 2024-08-01
+title: "We’ve decided to make less money [Part 2]"
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog_is_the_cheapest_e77c4ea4a5.jpg
+author:
+  - james-hawkins
+featuredImageType: full
+category: PostHog news
+---
+
+## TL;DR
+
+- Last week, we [slashed the cost of session replay](/blog/session-replay-pricing) by more than 50%, making PostHog the cheapest session replay tool in its class.
+
+- Today we're announcing Part 2, a change to our pricing that makes tracking events by anonymous users up to 80% cheaper.
+
+- We'll see an ~30% decline in product analytics revenue from the change, but it will unlock several new use cases, such as web analytics at massive scale, and tracking high-volume backend events, that were previously prohibitively expensive.
```

Comment:
```suggestion
- We'll see a ~30% decline in product analytics revenue from the change, but it will unlock several new use cases, such as web analytics at massive scale, and tracking high-volume backend events, that were previously prohibitively expensive.
```